packages/shared/src/components/checklist/SquadChecklistCard.tsx

```diff
@@ -60,7 +51,14 @@ const SquadChecklistCard = ({ squad }: { squad: Squad }): ReactElement => {
       className="flex w-full max-w-[21.5rem] justify-center rounded-none"
       onClose={onRequestClose}
     >
-      {checklistElement}
+      <ChecklistCard
+        title={
+          isDone ? 'Good job! you nailed it. 🥳' : 'Get started with squads'
+        }
+        description={`${totalStepsCount} simple steps to Squad greatness!`}
+        steps={steps}
+        className="max-w-full border-0"
```

Comment:
Don't you need `!border-0` here?
It might work without, but that's just luck of tailwind ordering then?

Reply:
Let me check whether we have to override to be sure.